### DISCUSSION TITLE
GitHub закрив деякі дискусії

### DISCUSSION THREAD

**Discussion by leofun01** [replace-org-ua/invites#13] [General]
GitHub закрив дискусії для команд.
На даний момент залишається можливість дискусій для окремих репо.
Дискусії організації в налаштуванях можна привязати до дискусій вибраного репо.
Старі дискусії ще доступні для допису. Але як створити нову дискусію для команди - я не знайшов.

<details>
	<summary>Список старих дискусій</summary>
	<ol>
		<li>
			<a href="https://github.com/orgs/replace-org-ua/teams/members/discussions/1">Дискусії в GitHub</a>
			(<a href="https://github.com/leofun01"><kbd>leofun01</kbd></a>)
		</li>
		<li>
			<a href="https://github.com/orgs/replace-org-ua/teams/members/discussions/2">Не вдається встановити з'єднання з replace.org.ua</a>
			(<a href="https://github.com/leofun01"><kbd>leofun01</kbd></a>)
		</li>
		<li>
			<a href="https://github.com/orgs/replace-org-ua/teams/members/discussions/3">Що ся знову трапило?</a>
			(<a href="https://github.com/fncamm"><kbd>fncamm</kbd></a>)
		</li>
		<li>
			<a href="https://github.com/orgs/replace-org-ua/teams/members/discussions/4">Запрошую на review дослідницького проекту.</a>
			(<a href="https://github.com/leofun01"><kbd>leofun01</kbd></a>)
		</li>
		<li>
			<a href="https://github.com/orgs/replace-org-ua/teams/members/discussions/5">Домен</a>
			(<a href="https://github.com/NaharD"><kbd>NaharD</kbd></a>)
		</li>
		<li>
			<a href="https://github.com/orgs/replace-org-ua/teams/members/discussions/6">Ban</a>
			(<a href="https://github.com/toplinden"><kbd>toplinden</kbd></a>)
		</li>
		<li>
			<a href="https://github.com/orgs/replace-org-ua/teams/members/discussions/7">Потестіть будь ласка мій сайт</a>
			(<a href="https://github.com/okineadev"><kbd>okineadev</kbd></a>)
		</li>
		<li>
			<a href="https://github.com/orgs/replace-org-ua/teams/members/discussions/8">Сайт лежить</a>
			(<a href="https://github.com/karmeljuk"><kbd>karmeljuk</kbd></a>)
		</li>
		<li>
			<a href="https://github.com/orgs/replace-org-ua/teams/members/discussions/9">Ось тепер форум лежить</a>
			(<a href="https://github.com/bvnflatliner"><kbd>bvnflatliner</kbd></a>)
		</li>
		<li>
			<a href="https://github.com/orgs/replace-org-ua/teams/members/discussions/10">Лежимо?</a>
			(<a href="https://github.com/pavloslav"><kbd>pavloslav</kbd></a>)
		</li>
	</ol>
</details>

Весь архів старих дискусій доступний як [json](https://github.com/orgs/replace-org-ua/teams/members/members/archived_team_posts.json).

**Comment by pavloslav** [comment]
Сайт помер?

**Reply by leofun01** [reply]
Так виглядає. Шкода.
Анастасія отримала інфо про проблему (як мінімум від мене) 2023-06-23. Мені не відомо хто займається цим (може ніхто).
З іншого боку, я не маю інфо про її стан і чи має вона можливість це пофіксити.
@pavloslav мабуть варто попросити доступ до FTP (я просити поки не буду бо тупо не маю часу).
Ну і сесюріті в форуму - просто сором.

**Reply by pavloslav** [reply]
Та я не певен, чи варто його воскрешати. Може, мігрувати на Reddit? Так, там власники нахабніють із API; але на replace узагалі API немає. Зате питаннями безпеки точно будуть профі займатися.

**Reply by leofun01** [reply]
> Може, мігрувати на Reddit?

Виявилося я вже маю там акаунт. Але Reddit намагався отримати доступ до куків не свого домена (а саме до [accounts.google.com](https://accounts.google.com/)). Коли йому це не вдалось, він намагався зробити fingerprint. Коли й це не вдалось, то залишив сторінку порожньою.

![зображення](https://github.com/replace-org-ua/invites/assets/5102759/d8c66238-2688-4a65-869c-cd2ad88f1d2b)
![зображення](https://github.com/replace-org-ua/invites/assets/5102759/84d153e6-f895-4e63-b9c5-77d88c5fc060)
Короче, не варіант. (<a href="https://www.youtube.com/watch?v=voD3akTcQrI&t=18">Як я намагався відкрити Reddit [видиво, коротка версія]</a>)

**Reply by marchelloUA** [reply]
Ось наґуґлив:

open source alternatives to phpBB are [Discourse](https://alternativeto.net/software/discourse/about/), [MyBB](https://alternativeto.net/software/mybb/about/), [Simple Machines Forum](https://alternativeto.net/software/simple-machines-forum/about/) and [NodeBB](https://alternativeto.net/software/nodebb/about/)

Це щоб хоча б якусь користь принести... Можу розгорнути щось з цього переліку в себе на vps аби спробувати що й до чого.

Нє ну я зарегався на місцевому форумі програмістів в країні де перебуваю, мовний бар'єр відсутній. Але декілька днів відчував що наче щось від мене забрали. Ватний доу юзати не дуже хочеться. 

Чи можна хоча б повішати цивілізоване повідомлення на сайті типу "ведуться роботи" чи щось таке?

**Reply by marchelloUA** [reply]
`[koala](https://replace.org.ua/user/931/) [Сьогодні 15:36:18](https://replace.org.ua/post/184120/#p184120) Хто це зробив, я вас питаю?`
Що серйозно - невідома причина збою, так само як і причина виправлення? А бекапи десь робляться?

**Reply by leofun01** [reply]
[воно живе [видиво]](https://www.youtube.com/watch?v=xos2MnVxe-c)

**Reply by marchelloUA** [reply]
`[koala](https://replace.org.ua/user/931/) [Сьогодні 21:30:42](https://replace.org.ua/post/184133/#p184133) Я так розумію, безпосередньо заражений файл було видалено антивірусом хостера, лишилося тільки посилання на нього.`
Тобто це веб-хостинг, а не vps? Отже, нема можливості встановити власний антивірус, наприклад clamav, аби налаштувати сканування щоночі? Тому сайт залежить від графіка запуску антивіруса і лежить днями поки не настане ця подія? А зв'язок з хостером хоча б є аби створити тікет на ручний запуск антивіруса?

**Comment by marchelloUA** [comment]
> Сайт помер?

`print("what is going on?")`
